routes/job.js

```diff
@@ -28,4 +32,72 @@ router.put('/', Utils.asyncRoute(async function (req, res, next) {
     res.send(room);
 }));
 
+router.get('/:jobId', Utils.asyncRoute(async function (req, res, next) {
+    const {jobId} = req.params;
+    const connector = new DbConnector();
+    try {
+        connector.connect();
+        const siteModel = await SiteModel.findOne({'rooms.jobs._id': mongoose.Types.ObjectId(jobId)}, {'rooms.jobs.$': true})
+        .sort({timestamp: 1}).exec();
+        const job = siteModel.rooms[0].jobs.find(job => job._id.equals(jobId));
+        
+        res.send(job);
+    } catch (e) {
+        console.error(e);
+        res.status(500).send(e);
+    }
+}));
+
+
+router.post('/analysis/:jobId', Utils.asyncRoute(async function (req, res, next) {
+    const {jobId} = req.params;
+    const {image, modelPrediction, analysisQueueId} = req.body
+    const connector = new DbConnector();
+    const timestamp = new Date().getTime();
+    const jobAnalysis = new JobAnalysis({
+        jobId: jobId,
+        timestamp: timestamp,
+        modelPrediction: modelPrediction,
+        image: image,
+    });
+    try {
+        connector.connect();
+
+        const siteModel = await SiteModel.findOne({'rooms.jobs._id': mongoose.Types.ObjectId(jobId)}, {'rooms.jobs.$': true});
+        const job = siteModel.rooms[0].jobs.find(job => job._id.equals(jobId));
+        
+        job.analysis.unshift(jobAnalysis);
+
+        siteModel.rooms[0]['lastAnalysedTimestamp'] = timestamp;
+        job.lastAnalysisId = analysisQueueId;
+        job.completed = modelPrediction >= 0.85;
+
+        await siteModel.save();
+
+        const analysisQueue = await AnalysisQueue.findById(analysisQueueId).exec();
+        const siteModel = await SiteModel.findOne(
```

Comment:
Duplicate declaration of variable

Reply:
Yea there is a larger bug in this endpoint im looking at. The analysisQueue will be set as completed even if only one job is analysed.  Ill push the variable name fix for now